src/components/Button/index.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import styles from "./index.module.css"
+
+const THEMES_VALUES = [
+  styles.redTheme,
+  styles.neonCarrotTheme,
+]
+
+export const THEMES = {
```

Comment:
How will this work if we have a bunch of components with a bunch of different themes?

Reply:
That's a great question, I don't know how to answer that yet. Maybe the themes can be abstracted, as they should be the same for all?